### DISCUSSION TITLE
Follow and Silent System.

### DISCUSSION THREAD

**Discussion by Glustiksas** [diamond-rip/Eden#190] [Ideas]
You could create a mechanic that would help a server staff member who uses this wonderful plugin, an incredibly interesting option for those who love to moderate and also for being extremely useful than always giving (/spectate) all the time or giving (/disguise) in a core plugin to hide yourself or your nickname. (An example of (/silent) below)

![2023-03-17_02 54 37](https://user-images.githubusercontent.com/127461681/226696379-a5e22922-9af3-4fe3-979b-1d326acc338c.png)

**Comment by Molarik** [comment]
This idea is extremely perfect and I think it should be implemented in the plugin, but I beg you to add this... I was just passing by and saw this interesting thing and decided to comment on it... please add this mechanic! 🙏🙏🙏

**Reply by Glustiksas** [reply]
Yes man, but i have good example for it.. 🤔

[FollowACTION.txt](https://github.com/diamond-rip/Eden/files/11032075/FollowACTION.txt)
[FollowSTATUS.txt](https://github.com/diamond-rip/Eden/files/11032076/FollowSTATUS.txt)

**Comment by GoodestEnglish** [comment]
Could you please describe what does /slient do?

**Reply by Stainded** [reply]
@GoodestEnglish, 

It puts you into an anonymous mode when watching some event or some match, for example

not using silent:
Winner: Player1
Loser: Player2
Spectator (2): Spectator1<YOU_YOU>, Spectator2<Other_player>


Using Silent: 

Winner: Player1
Loser: Player2
Spectator (1):  Spectator1<Other_Player>

##You are watching but you are in hidden or anonymous mode (SILENT -> Does not display the status that you are watching the match and does not send the message "YOU is now spectating the match"

**Reply by Glustiksas** [reply]
@GoodestEnglish, Stained is right... it really is that system and Stained thanks for explaining it to him somehow

**Reply by GoodestEnglish** [reply]
Interesting concept, didn't know this command existed for most of the servers. Guess I'm old now 😅

Jokes aside, I will mostly add this feature with high priority. Thanks for the suggestion.

**Reply by Glustiksas** [reply]
Thank you very much, I will keep you posted on new things but if you need my help with the project in any way I will always be available! ;)

**Reply by Stainded** [reply]
@Glustiksas, Now I'm confused whether he replied to me or to you :v
